clinica/pipelines/pet_volume/pet_volume_utils.py

```diff
@@ -70,48 +140,81 @@ def apply_binary_mask(image, binary_mask):
     return masked_image_path
 
 
-def create_pvc_mask(tissues):
+def create_pvc_mask(tissues: list) -> str:
+    """Create a pvc mask from tissue list.
 
+    Parameters
+    ----------
+    tissues : list
+        List of paths to tissue Nifti1Images. Must be non-empty.
+
+    Returns
+    -------
+    out_mask : str
+        Path to the resulting mask Nifti1Image.
+    """
     from os import getcwd
     from os.path import join
 
     import nibabel as nib
     import numpy as np
 
-    if len(tissues) == 0:
-        raise RuntimeError(
-            "The length of the list of tissues must be greater than zero."
-        )
-
-    img_0 = nib.load(tissues[0])
-    shape = img_0.get_fdata(dtype="float32").shape
-    background = np.zeros(shape=shape)
+    from clinica.pipelines.pet_volume.pet_volume_utils import _load_tissues  # noqa
 
+    background, affine, header = _load_tissues(tissues)
+    shape = background.shape
     shape += tuple([len(tissues) + 1])
     data = np.empty(shape=shape, dtype=np.float64)
-
-    for i in range(len(tissues)):
-        image = nib.load(tissues[i])
+    for i, tissue in enumerate(tissues):
+        image = nib.load(tissue)
         data[..., i] = np.array(image.get_fdata(dtype="float32"))
-        background = background + image.get_fdata(dtype="float32")
-
     background = 1.0 - background
     data[..., len(tissues)] = np.array(background)
-
     out_mask = join(getcwd(), "pvc_mask.nii")
-    mask = nib.Nifti1Image(data, img_0.affine, header=img_0.header)
+    mask = nib.Nifti1Image(data, affine, header=header)
     nib.save(mask, out_mask)
     return out_mask
 
 
-def pet_pvc_name(pet_image, pvc_method):
+def pet_pvc_name(pet_image: str, pvc_method: str) -> str:
+    """Build the name for the PET PVC interface.
+
+    Parameters
+    ----------
+    pet_image : str
+        Path to the PET scan file.
+
+    pvc_method : str
+        Name of the PVC method. This will be concatenated
+        with the `pet_image` filename.
+
+    Returns
+    -------
+    pet_pvc_path : str
+        Name for the PET PVC interface.
+    """
```

Comment:
Looks like this function could have an example ! (since it's not dealing with images)

Reply:
Never hurts indeed ! I added an example in [2bc4ed7](https://github.com/aramis-lab/clinica/pull/729/commits/2bc4ed7b2292f2ffc83c164d3b9cd393ab623b10)